resources/META-INF/includes/VimExCommands.xml

```diff
@@ -32,10 +32,10 @@
     <vimExCommand implementation="com.maddyhome.idea.vim.ex.handler.MarkHandler" names="ma[rk],k"/>
     <vimExCommand implementation="com.maddyhome.idea.vim.ex.handler.MarksHandler" names="marks"/>
     <vimExCommand implementation="com.maddyhome.idea.vim.ex.handler.MoveTextHandler" names="m[ove]"/>
-    <vimExCommand implementation="com.maddyhome.idea.vim.ex.handler.NextFileHandler" names="n[ext]"/>
+    <vimExCommand implementation="com.maddyhome.idea.vim.ex.handler.NextFileHandler" names="n[ext],bn"/>
```

Comment:
This is not the full command. It should be `bn[ext]`. The same works for other commands.

Reply:
fixed it